app/gui/view/debug_scene/component-group/src/lib.rs

```diff
@@ -40,6 +42,11 @@ use list_view::entry::AnyModelProvider;
 // =================
 
 const COMPONENT_GROUP_COLOR: color::Rgba = color::Rgba::new(0.527, 0.554, 0.18, 1.0);
+/// The selection animation is faster than default one because of the increased spring force.
+const SELECTION_ANIMATION_SPRING_FORCE: f32 = 30_000.0;
```

Comment:
Leave here a multiplier intead. So if we change the spring force globally thsi will update automatically.

Reply:
Done